### PR TITLE
Toolchains setup scripts for centos8.

### PR DESCRIPTION
**Pull request by matrixji**
issue: #8110

Signed-off-by: Ji Bin <matrixji@live.com>

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -82,6 +82,21 @@ $ wget -q https://boostorg.jfrog.io/artifactory/main/release/1.65.1/source/boost
 
 ```
 
+- CentOS 8
+
+```shell
+# Enable powertools
+$ sudo dnf config-manager --set-enabled powertools
```
Comment (LoveEachDay) [inline DEVELOPMENT.md:89]: You may need to install config-manager first, or check whether config-manager is installed.

Comment (matrixji) [reply DEVELOPMENT.md:89]: Correct, in some minimal Installation, `dnf-plugins-core` may missing.

**Comment by XuanYang-cn** [comment]
/hold

Thanks for the contribution, but we found a `ld` error for openblas while running cpp-tests in centOS8

This PR will be held until I confirm the same error also occurs in Ubuntu. 

See #8143 for updates. 

Anyway, this's a bug and will be fixed soon.

**Comment by LoveEachDay** [comment]
The changes looks good to me. But we have an issue due to the openblas compile output a different directory, rather than `internal/core/output/lib`. 
We'll fix this issue first before merge this commit.

**Comment by matrixji** [comment]
I see, after erasing the system's openBLAS, also reproducing it.

**Comment by XuanYang-cn** [comment]
Yes it's unique on centOS. Will fix it soon

**Pull request closed without merging**